src/metabase/analytics/stats.clj

```diff
@@ -141,6 +141,8 @@
    :embedding_app_origin_set             (boolean
                                           #_{:clj-kondo/ignore [:deprecated-var]}
                                           (setting/get :embedding-app-origin))
+   ;; We no longer add "localhost:*" as a default origin as of Metabase 56, as it is always allowed,
+   ;; but we still filter it out in stats for compatibility with migrated instances.
```

Comment:
I'll keep the `localhost:*` filtering on stats as people will still have the old `localhost:*` defaults in their instances.